app/controllers/FaciaContentApiProxy.scala

```diff
@@ -51,6 +51,10 @@ class FaciaContentApiProxy(val deps: BaseFaciaControllerComponents)(implicit ec:
 
 
     wsClient.url(url).withHttpHeaders(getPreviewHeaders(url): _*).get().map { response =>
+
+      if (response.status != OK) {
```

Comment:
Yes, this is what we want - I didn't want to change the handling of errors because I thought it might be happening in the old app as well, users are just not seeing it because we keep polling capi for new articles. We have some more information now so I'll be able to look into this today.

Reply:
@regiskuckaertz @tomrf1 Should be able to investigate the CAPI logs for you at the time you are seeing a 403/500 error code and understand what's happening on the Concierge side.

Reply:
I have had a look - it seems to be coming from api gateway. So if the logging can't tell us more then we'll have to ask AWS for help